tests/VObject/EmClientTest.php

```diff
@@ -48,8 +51,13 @@ public function testParseTz(): void
 END:VEVENT
 END:VCALENDAR';
 
+        /** @var VCalendar<int, mixed> $vObject */
```

Comment:
can we define a more precise return-type instead?

https://github.com/sabre-io/vobject/blob/7c585e61435306d4030f7e97260d6292892173ba/lib/Reader.php#L43

`@var` has the problem, that we need to declare it on every call-site